lib/active_merchant/billing/gateways/merchant_warrior.rb

```diff
@@ -161,6 +164,16 @@ def verification_hash(money, currency)
         )
       end
 
+      def void_verification_hash(transaction_id)
+        Digest::MD5.hexdigest(
+          (
+            @options[:api_passphrase].to_s +
```

Comment:
You mentioned `the required verification hash for void transactions should consist of the original transaction ID`. Is it fine also adding in the `api_passphrase` and `merchant_uuid` like what's being done here? I assume so, but figured I'd ask.

Reply:
Yes, those fields are also needed. This was the response from Merchant Warrior to our customer about how to build the hash for a void transaction:
```
Note, the hash is calculated by: strtolower(md5(md5(api_passphrase).uuid.transactionid))
```

Reply:
Ah, great to know.